.github/workflows/ci-goreleaser.yaml

```diff
@@ -44,4 +51,6 @@ jobs:
         uses: goreleaser/goreleaser-action@v3
         with:
           version: latest
-          args: --snapshot --rm-dist --timeout 1h
+          args: --snapshot --rm-dist --skip-sign --timeout 2h
```

Comment:
What happens when `--skip-sign` is set, but `COSIGN_EXPERIMENTAL: true` is there?

Reply:
nothing

Reply:
removed